src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "core.basesyntax.Ball{"
```

Comment:
The string representation of the class in the toString() method should not include the full package name. You can simply use 'Ball' instead of 'core.basesyntax.Ball'.